test/integ/test_put_get.py

```diff
@@ -646,3 +648,21 @@ def test_put_special_file_name(tmp_path, conn_cnx):
             ).fetchall()
             cur.execute(f"select $1, $2, $3 from  @{stage_name}")
             assert cur.fetchone() == ("1", "2", "3")
+
+
+@pytest.mark.skipolddriver
+def test_get_empty_file(tmp_path, conn_cnx):
+    test_file = tmp_path / "data.csv"
+    test_file.write_text("1,2,3\n")
+    stage_name = random_string(5, "test_get_empty_file_")
+    with conn_cnx() as cnx:
+        with cnx.cursor() as cur:
+            cur.execute(f"create temporary stage {stage_name}")
+            filename_in_put = str(test_file).replace("\\", "/")
+            cur.execute(
+                f"PUT 'file://{filename_in_put}' @{stage_name}",
+            ).fetchall()
```

Comment:
Results are not actually used
```suggestion
            )
```